pkg/collector/metric/node_metric.go

```diff
@@ -61,6 +62,7 @@ type NodeMetrics struct {
 	TotalEnergyInGPU      *types.UInt64StatCollection
 	TotalEnergyInOther    *types.UInt64StatCollection
 	TotalEnergyInPlatform *types.UInt64StatCollection
+	TotalEnergyInRedfish  *types.UInt64StatCollection
```

Comment:
I was thinking, if redfish, is a kind of platform energy api, then we may don't need a new var as `TotalEnergyInRedfish` but reuse `TotalEnergyInPlatform`?
as redfish api is a type of Platform API?

Reply:
I agree.

Also, if redfish is enabled, we don't need to collect the platform power from other sources.

Reply:
sounds good, working on the refactoring